src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -367,4 +370,78 @@ public boolean contains(String key) {
 
         return mPrefs.contains(key);
     }
+    
+    private TokenCacheItem getFamilyRefreshTokenItemForUser(String userId) {       
```

Comment:
hm, then we should have the caller provide us the authority. It doesn't make sense to query the cache just for the purpose of getting authority.